spec/recurly/add_on_spec.rb

```diff
@@ -85,5 +122,48 @@
       add_on.percentage_tiers.first.tiers.count.must_equal 2
       add_on.percentage_tiers.first.tiers.first.must_be_instance_of PercentageTier
     end
+
+    it "returns RevRec attributes" do
+      plan = Plan.find 'gold'
+      add_ons = plan.add_ons
+
+      add_on = add_ons.first
+      add_on.must_be_instance_of AddOn
+      add_on.liability_gl_account_id.must_equal "udexyr9hjgkc"
+      add_on.revenue_gl_account_id.must_equal "uelq7rzkydlu"
+      add_on.performance_obligation_id.must_equal "6"
+    end
+  end
+
+  describe "create" do
+    it "must create a new add-on" do
+      stub_request(:post, "/plans")
+      plan = Plan.find 'gold'
+      add_on = AddOn.new(
+        add_on_code: 'pink_sweaters',
+        name: 'Pink Sweaters',
+        revenue_schedule_type: 'evenly',
+        unit_amount_in_cents: 200,
+        add_on_type: 'usage',
+        optional: false,
+        usage_type: 'price',
+        liability_gl_account_id: 'uf0jwj5zhclg',
+        revenue_gl_account_id: 'uf0jwincednb',
+        performance_obligation_id: '1'
+      )
+      add_on.must_be_instance_of AddOn
+      add_on.name.must_equal 'Pink Sweaters'
+    end
+  end
```

Comment:
We are missing a call to `.create` in this example.

Reply:
Updated